fenix/app/src/main/java/org/mozilla/fenix/compose/LinkText.kt

```diff
@@ -95,15 +97,21 @@ fun LinkText(
         modifier = Modifier.clickable(
             enabled = context.isScreenReaderEnabled,
             role = Role.Button,
-            onClickLabel = linkTextState.text,
-            onClick = { linkTextState.onClick(linkTextState.url) },
+            onClickLabel = linkTextStates.first().text,
+            onClick = {
+                linkTextStates.first().let {
```

Comment:
Let's use `firstOrNull` since users can pass it `emptyList()`, would be nice to have a `NonEmptyList` data structure here.